contracts/protocol/tokenization/NTokenMoonBirds.sol

```diff
@@ -32,11 +33,15 @@ contract NTokenMoonBirds is NToken, IMoonBirdBase {
         // Intentionally left blank
     }
 
+    function getXTokenType() external pure override returns (XTokenType) {
```

Comment:
I think we can simplify it as `XTokenType()` because `name` and `symbol` doesn't have `get` prefix

Reply:
it's ok. we also have many get function style.